WL-4237 Removed gaps between paragraphs in Syllabus tool.

ouit0408 opened the pull request:
Added margin for <p> tag in syllabus.css to remove extra space when text is
added through CKEditor.

buckett commented:
Where was the original extra space coming from?

buckett commented:
I think it's coming from this and affects more than just `<p>` tags (`<div>` etc).
![screen shot 2016-02-26 at 14 41 20](https://cloud.githubusercontent.com/assets/5921/13355199/0bbcef42-dc97-11e5-9dc1-4406d732e30f.png)

ouit0408 commented:
it looks like in sakai syllabus tool, for textarea element deliberately 'editable-pre-wrapped ' class is added - ref SAK-23451 . But the code in jqueryui-editable.js which was using this styling has been commented out. Shall i remove this class from textarea ?

buckett commented:
Just chatted about this in the office and it looks like the `editable-pre-wrapped` class is only used to preserve paragraphs and newlines. As we're editing HTML directly we don't need to worry about this. The cleanest fix is to just override the CSS as this is easier to manage in the long run, rather than trying to change the JS and merge it as newer versions of the library come out.

ouit0408 commented:
have commented the styling for editable-pre-wrapped class and have updated the PR.

buckett commented on syllabus-app/src/webapp/syllabus/css/jqueryui-editable.css:
```diff
@@ -136,7 +136,7 @@
 }
 
 .editable-pre-wrapped {
-   white-space: pre-wrap;
```
Much better off to override this CSS iin syllabus.css as the jquery-editable.css is taken as-is from an upstream source and if a new version of the library gets pulled in this change will probably get lost.

ouit0408 committed:
WL-4237 Removed gaps between paragraphs in Syllabus tool.

Overriding the styling for white-space in syllabus.css as it is
better to change in css than js file for future versions of the library.

ouit0408 commented:
made the change and have updated the PR
